### ISSUE TITLE
Separate Hotels tab

### ISSUE THREAD

**Issue by mikebroom** [CardiffBros/spoons-tracker-web#66]
There are a few that are doing the hotels - a separate hotel list/ visit section would be useful. Would suggest keeping it separate to avoid cluttering with pubs...

**Comment by nottoast** [comment]
I like the idea, we could have a separate list of 'stayed in' (as well as the current 'visited'), so people could mark off hotels as one or both. More of a v2 feature this though I think

**Comment by mikebroom** [comment]
Absolutely, definitely not a priority as suspect the numbers counting these so far is quite small.
